CONTRIBUTING.md

```diff
@@ -1,57 +1,64 @@
-# Contributing to Archive of Our Own
+# Contributing to OTW-Archive
+
+## Reporting bugs
+
+We maintain a [JIRA issue tracker](https://otwarchive.atlassian.net) for developers,
+and a [list of Known Issues](https://archiveofourown.org/known_issues) for
+[Archive of Our Own](https://archiveofourown.org) users, neither of which are
+publicly editable.
+
+If you need help using the site, or want to report an issue you have found,
+please [contact the AO3 Support team](https://archiveofourown.org/support).
 
-## Contributing code
 
-### Report a bug
+## Reporting security issues
 
-We have a [list of known issues](https://archiveofourown.org/known_issues).
-If you have a problem that you don't see on that list, please send a bug report
-via the [Support and Feedback Form](https://archiveofourown.org/support).
+Please see [SECURITY.md](https://github.com/otwcode/otwarchive/blob/master/SECURITY.md).
 
-### Pick an issue to work on
 
-You will find our [list of approved issues on JIRA](https://otwarchive.atlassian.net/projects/AO3/issues).
+## Suggesting new features
```

Comment:
🍷

Reply:
Should we move this section lower?

Reply:
Hmm. Yeah, sure, let's put it after "Updating documentation."